src/result_processor.c

```diff
@@ -400,10 +443,6 @@ static int rpsortNext_innerLoop(ResultProcessor *rp, SearchResult *r) {
           return RS_RESULT_ERROR;
         }
       }
-    } else if (self->sortbyType == SORTBY_DISTANCE){
```

Comment:
`sortbyType` enum can be removed.
Remove also from signature of function `RPSorter_NewByFields`. Look [here](https://github.com/RediSearch/RediSearch/pull/2401/files) for change. 

It was added for the old implementation.

Reply:
I thought we need it for sortby_score, Ill remove it